### ISSUE TITLE
analyzer: Identify a player's role in a match

### ISSUE THREAD

**Issue by kvahuja** [vainglorygame/meta#36]
So for role a user played - this is how i think:
1. Captain: identify based on utility icon. It has to be "100%" or if captain had a hybrid like Cath say builds Aftershock or SG or some. 
2. Once we know a participant is not a Captain, we then have to identify if a player was a carry of jungler. While the game classifies a Laner as carry and Jungler to peel, technically some players play in Juingle but act as carry while Laners end up peeling. See L3oN's game play in V8 this weekend. He and JustMan moved across lane and jungler but theur roles were clear - Justman was Peeling while L3oN was suppsed to carry. This distinction is required as well IMHO.
2.1 So we can use CS to identify who's playing where but we need a network to identify who's really carry and who's peeling
----
In summary (TLDR;)
 - items to identify a captain. Further break into sub-roles - Tank / Semi-Tank??
 - CS to identify a role - Laner / Jungler and then in each of those identify carry vs/ peeler
(we have to explain these terms). Plus need to check with @odds if there are better naming conventions that can be used.
---
To calculate I am fine if we want to train a DNN. If i was to think this with simple SQL, i would use Items (captains), Minions (for 1st level breakup) and then Kills or items to identify Carry vs. Peeler.

**Comment by schneefux** [comment]
Do items identify a captain? Is someone a captain who has a fountain but never uses it?
In my opinion, usage of vision is vital to the role.

**Comment by kvahuja** [comment]
so you are right, it's absolutely key but not for identification of a role but how effective a player was in that role.

```you pick a role and play it vs. how effectively you play it. ```

**Comment by schneefux** [comment]
see also #78
